setup.py

```diff
@@ -150,13 +149,14 @@ def run(self):
     extras_require["nodocs"].extend(reqs)
 
 install_requires = [
-    "appdirs>=1.4.3,<=1.4.4 ",
+    "PyYAML>=3.12,<=5.3.1",
```

Comment:
Did you use isort for this? I wonder if case-insensitive sort would be better. At least I could see myself searching for pyyaml and being confused when it's not with the other `py*` entries. But just a minor detail

Reply:
Fair point. Used case-insensitive sort.